src/sharedData/components/SharedDataCard.js

```diff
@@ -59,11 +84,7 @@ const SharedFilesCard = ({ group }) => {
               </Typography>
             </>
           )}
-          <Button
-            variant="outlined"
-            component={RouterLink}
-            to={`/groups/${group.slug}/upload`}
-          >
+          <Button variant="outlined" onClick={onUploadClick}>
```

Comment:
@paulschreiber I don't think that name is better, reading it, it makes me think it will handle the upload process, and that is not correct, maybe it can be `handleUploadButtonClick` but only `handleUpload` is confusing

Reply:
I'm ok with `onAction` name pattern in these cases.